nat-lab/tests/utils/command_grepper.py

```diff
@@ -49,7 +51,7 @@ async def check_exists(
             print(f"Process exec error: {e}")
             raise
         except TimeoutError as e:
-            print(f"Timeout error: {e}")
+            print(f"Timeout error: {e}, last stdout: {self._last_stdout}")
```

Comment:
nit: maybe we can print `_last_stdout` on all exceptions, not only timeout

Reply:
Yeah, I though about that, decided not, but what the hell, let's do it :smile:

Reply:
it will not hurt to have extra logs anyway :laughing: